less/components/header.less

```diff
@@ -36,31 +36,32 @@
       display: inline-block;
       color: inherit;
       font-weight: 700;
+      height: 18px;
```

Comment:
Thanks for the feedback, I'll get on these changes. I think that issue with safari may be what the `transform: translateY(0px);` line was for so I'll drop that back in the hover state. Unfortunately I am on Linux and do not know how I can test this in Safari/Mac OS to be sure it is working properly.

The left/right padding on the black box should not shift the nav items because the hover and the nav items are on a different z-index so that should not be too much trouble.